starcounter-include.html

```diff
@@ -249,24 +249,30 @@
         StarcounterIncludePrototype._notifyPath = function(path, value) {
             if (path.indexOf("partial.") === 0 || path.indexOf("viewModel.") === 0) {
                 if (
-                    path === "partial.Html" ||
-                    // path.match(/^partial\.[^.]+\.Html$/)
-                    (path.indexOf('.Html') === path.length - 5) && path.slice(8, -5).indexOf('.') === -1
+                    // path === "viewModel.Html" || // covered by  viewModel._whatever_
+                    // path.match(/^viewModel\.[^.]+\.Html$/)
+                    (path.indexOf('.Html') === path.length - 5) && path.slice(10, -5).indexOf('.') === -1
                 ) {
                     this.partialChanged(this.partial);
                 } else if (
-                    path === "viewModel.Html" ||
-                    // path.match(/^viewModel\.[^.]+\.Html$/)
-                    (path.indexOf('.Html') === path.length - 5) && path.slice(10, -5).indexOf('.') === -1
+                    // path === "partial.Html" || // covered by  partial._whatever_
+                    // path.match(/^partial\.[^.]+\.Html$/)
+                    (path.indexOf('.Html') === path.length - 5) && path.slice(8, -5).indexOf('.') === -1
                 ) {
                     this.partialChanged(this.partial);
+                } else
+
+                // if viewModel|partial._whatever_, but not _ver - scope changed
+                if(path.split('.').length === 2 && path.indexOf('_ver#s') === -1 && path.indexOf('_ver#c$') === -1){
+                    this.partialChanged(this.partial);
```

Comment:
>  Could you pls organize it more commonly?

What is then the common way to describe the condition? I usually made it before `if` statement


> Isn't `path.split('.').length` too broad? This will catch viewModel.People_0.FirstName$ etc

`path.split('.').length` will, but `path.split('.').length === 2` that is used above won't.